docs/cql/ddl.rst

```diff
@@ -235,7 +235,7 @@ For instance::
 
 
   ALTER KEYSPACE Excelsior
-   WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 4};
+   WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor' : 4};
```

Comment:
same comment as in https://github.com/scylladb/scylladb/pull/15856/files#r1372967154